usaspending_api/recipient/tests/integration/test_state.py

```diff
@@ -2,12 +2,12 @@
 import datetime
 import decimal
 
-# Core Django imports
+import pytest
+from model_bakery import baker
 
+# Core Django imports
 # Third-party app imports
 from rest_framework import status
-from model_bakery import baker
```

Comment:
I only complain about this because you moving those around makes that comment not hold as much value now.

Any reason to move those from under the `# Third-party app imports`

Reply:
This is my linter moving around imports. I'm not sure why it moved the `model_baker` import. I'll undo this import move.